.github/workflows/ci.yml

```diff
@@ -335,7 +348,11 @@ jobs:
       - name: Run smoke-test-k8-tls-root
         timeout-minutes: 5
         run: |
-          make FLUVIO_BIN=./fluvio TEST_BIN=./flv-test smoke-test-k8-tls-root
+          make FLUVIO_BIN=./fluvio TEST_BIN=./flv-test UNINSTALL=noclean smoke-test-k8-tls-root
+
+      - name: Clean minikube
+        run: |
```

Comment:
Should this have `if: always()` so that even if the test fails it still cleans up minikube?

Reply:
Yes, updated both local and k8 cluster tests